wave_containers/include/wave/containers/measurement.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef WAVE_CONTAINERS_MEASUREMENT_HPP
+#define WAVE_CONTAINERS_MEASUREMENT_HPP
+
+#include <chrono>
+
+namespace wave {
+
+using TimeType = std::chrono::steady_clock::time_point;
```

Comment:
Not a fixed decision, just time_point is more convenient to use (it has math and comparison operators already defined) and explicitly represents a time (as opposed to a duration). It could change it or make it flexible like the sensor and value types.